packages/prisma/seed.ts

```diff
@@ -604,6 +608,24 @@ async function main() {
       },
     ]
   );
+  if (team) {
+    for (const eventType of team.eventTypes) {
+      await prisma.eventType.update({
+        where: {
+          id: eventType.id,
+        },
+        data: {
+          users: {
```

Comment:
As users weren't set, no timeslots were selectable in the seeded team events. They work now.